src/reducers/change-fixture.js

```diff
@@ -0,0 +1,10 @@
+var CHANGE_FIXTURE = require('../actions/change-fixture.js').type;
+
+module.exports = function(state = {}, action) {
+  switch (action.type) {
+    case CHANGE_FIXTURE:
+      return action.data || {};
```

Comment:
There's no `||` in reducer because `reducers` should be __pure__ functions. You should have an empty `payload` for data.